include/world_builder/utilities.h

```diff
@@ -457,11 +457,13 @@ namespace WorldBuilder
      * @param position_in_natural_coordinates_at_min_depth the current position in natural_coordinates
      * @return The content of the file.
     */
-    std::pair<double, double>
+    std::vector<double>
     calculate_ridge_distance_and_spreading(std::vector<std::vector<Point<2>>> mid_oceanic_ridges,
                                            std::vector<std::vector<double>> mid_oceanic_spreading_velocities,
                                            const std::unique_ptr<WorldBuilder::CoordinateSystems::Interface> &coordinate_system,
-                                           const Objects::NaturalCoordinate &position_in_natural_coordinates_at_min_depth);
+                                           const Objects::NaturalCoordinate &position_in_natural_coordinates_at_min_depth,
+    std::vector<std::vector<double>> subducting_plate_velocities = {{0.0}},
+    std::vector<double> ridge_migration_times = {0.0});
```

Comment:
I don't think this is a good solution. Vectors generally should be passed in as references. Although default paramaters can be nice, especially with backwards compatibility for interfaces, I don't think they should be used here, given that this function is only used internally. I think in this case it is better to update all places where it is used with an reference to an actual vector.